SourceKitStressTester/Sources/StressTester/SourceKitDocument.swift

```diff
@@ -593,6 +594,9 @@ class SourceKitDocument {
       }
       tree = try Parser.parse(source: source)
     }
+    if let foldedTree = try? OperatorTable.standardOperators.foldAll(tree).as(SourceFileSyntax.self) {
```

Comment:
If you swallow all of the errors produces by folding, with, e.g.,

```swift
OperatorTable.standardOperators.foldAll(tree) { error in }
```

then it'll provide better test coverage, because it should always produce a fully-folded tree that round-trips, even in the presence of errors.